### DISCUSSION TITLE
Backup a cloud version

### DISCUSSION THREAD

**Discussion by sergey-vin** [Budibase/budibase#7570] [General]
Hi!

Is there any way to backup a cloud version of the app?
In case someone messes up with it, I want to have an ability to restore some snapshot.
I see people are referring to some JSON exports in similar discussions, but I couldn't find any user manual on how to do a backup.
Thanks.

**Comment by melohagan** [comment]
Hey @sergey-vin 

If you want to backup your cloud app you can export it: https://docs.budibase.com/docs/portal#export-app

As a side note, if you are running Self-Hosted then you can do a full backup snapshot with the CLI:  https://docs.budibase.com/docs/backups

**Reply by sergey-vin** [reply]
Thanks, that worked!

**Reply by BennaceurHichem** [reply]
Should this export could be used to pass from cloud to Self hosted?
